README.md

```diff
@@ -161,6 +161,32 @@ npm run test:integration
 
 N.B. The integration tests rely on there being a test server available on localhost:8080.
 
+## Deployment
+
+When code is merged into the master branch an Azure pipline will be started. If the pipeline passes it will produce an artifact.
+
+Take a copy of the artifact and rename it from main.js to cookie-consent.js.
```

Comment:
This reads as an instruction, but it's part of the automated release pipeline. Suggest changing the wording slightly to make this clear.